src/client/Microsoft.Identity.Client/AppConfig/AuthorityType.cs

```diff
@@ -4,19 +4,28 @@
 namespace Microsoft.Identity.Client
 {
     /// <summary>
+    /// Enumeration for the AuthorityTypes
     /// </summary>
     internal enum AuthorityType
     {
         /// <summary>
+        /// Azure Active Directory
         /// </summary>
         Aad,
 
         /// <summary>
+        /// ADFS authority
         /// </summary>
         Adfs,
 
         /// <summary>
+        /// For perosnal/social accounts
         /// </summary>
-        B2C
+        B2C, 
+
+        /// <summary>
+        /// For DSTS only

```

Comment:
If comments do not add any value / information, better to remove them (at least for internal types)